internal/fakes/fake_images.go

```diff
@@ -15,8 +16,10 @@ import (
 	h "github.com/buildpacks/pack/testhelpers"
 )
 
-func NewFakeBuilderImage(t *testing.T, tmpDir, name string, stackID, uid, gid string, metadata builder.Metadata, bpLayers dist.BuildpackLayers, order dist.Order) *fakes.Image {
-	fakeBuilderImage := fakes.NewImage(name, "", nil)
+type FakeImageCreator func(string, string, imgutil.Identifier) *fakes.Image
```

Comment:
We should name the input variables in the function type, to help when creating new ones.